packages/commonwealth/client/scripts/views/pages/new_snapshot_proposal/SnapshotSpaceSelectorModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  CWModalBody,
+  CWModalFooter,
+  CWModalHeader,
+} from 'views/components/component_kit/new_designs/CWModal';
+import { CWButton } from '../../components/component_kit/new_designs/CWButton';
+
+type SnapshotSpaceSelectorModalProps = {
+  snapshotSpacesArray: string[];
+  onModalClose: () => void;
+  handleSpaceSelection: (space: string) => void;
+};
+const SnapshotSpaceSelectorModal = ({
+  snapshotSpacesArray,
+  onModalClose,
+  handleSpaceSelection,
+}: SnapshotSpaceSelectorModalProps) => {
+  return (
+    <div className="SnapshotSpaceSelectorModal">
+      <CWModalHeader
+        label="Please select a Snapshot space"
+        onModalClose={onModalClose}
+      />
+      <CWModalBody>
+        <div>
+          {snapshotSpacesArray.map((space, key) => (
+            <CWButton
+              label={space}
+              key={key}
+              onClick={() => handleSpaceSelection(space)}
+            ></CWButton>
+          ))}
+        </div>
+      </CWModalBody>
+      <CWModalFooter> </CWModalFooter>
```

Comment:
is it necessary if it is empty?

Reply:
@masvelio I put it there because I was anticipating a redesign from Jess/Jared, and because it easily gave some padding on the bottom. I can add padding in with css for the time being if you prefer.